eventarc-events/EventListGenerator/Program.cs

```diff
@@ -60,9 +60,9 @@ private static void AddHeader(StreamWriter file)
         {
             file.WriteLine("# Events supported by Eventarc\n");
             file.WriteLine("The following is a list of the events supported by Eventarc.\n");
-            file.WriteLine("- Using Pub/Sub");
-            file.WriteLine("- Using Cloud Audit Logs");
             file.WriteLine("- Directly from a Google Cloud source");
+            file.WriteLine("- Using Cloud Audit Logs");
+            file.WriteLine("- Using Pub/Sub");

```

Comment:
nit: there are less providers using pub/sub, so I would prefer having it before CAL.